consumer/src/main/kotlin/streams/kafka/KafkaEventSink.kt

```diff
@@ -75,19 +77,20 @@ class KafkaEventSink(private val config: Map<String, String>,
     }
 
     override fun start() { // TODO move to the abstract class
-        val streamsConfig = StreamsSinkConfiguration.from(config)
+        val streamsConfig = StreamsSinkConfiguration.from(config, db.databaseName())
         val topics = streamsTopicService.getTopics()
         val isWriteableInstance = Neo4jUtils.isWriteableInstance(db)
         if (!streamsConfig.enabled) {
             if (topics.isNotEmpty() && isWriteableInstance) {
-                log.warn("You configured the following topics: $topics, in order to make the Sink work please set the `${StreamsSinkConfigurationConstants.STREAMS_CONFIG_PREFIX}${StreamsSinkConfigurationConstants.ENABLED}` to `true`")
+                log.warn("You configured the following topics: $topics, in order to make the Sink work please set the `${StreamsConfig.SINK_ENABLED}` to `true`")
```

Comment:
make it copy/pasteable.   "Please set ${StreamsConfig.SINK_ENABLED}=true"

Reply:
Done